src/strategy/remote-addresss-strategy.ts

```diff
@@ -10,7 +11,16 @@ export class RemoteAddressStrategy extends Strategy {
         if (!parameters.IPs) {
             return false;
         }
-        const ipList = parameters.IPs.split(/\s*,\s*/);
-        return ipList.includes(context.remoteAddress);
+        for (const range of parameters.IPs.split(/\s*,\s*/)) {
+            if (!ip.isV6Format(range)) {
+                if (ip.cidrSubnet(range).contains(context.remoteAddress)) {
+                    return true;
+                }
+            }
+            if (range === context.remoteAddress) {
```

Comment:
Minor: Maybe this should the the if `if` statement. Feels like this could be faster than the range check?